ee/backend/app/src/main/kotlin/io/tolgee/ee/component/slackIntegration/SlackExecutor.kt

```diff
@@ -62,18 +61,20 @@ class SlackExecutor(
       }
```

Comment:
Above ^^, You should find the saved messages in one request using in operator in one SQL requests instead of iterating like this. Also the property should be named `savedMessages` and then when you iterate over it you got single `savedMessage`.